Core/Importers/KredoTransactionsImporter.cs

```diff
@@ -0,0 +1,64 @@
+﻿using iText.IO.Font;
+using iText.Kernel;
+using iText.Kernel.Font;
+using iText.Kernel.Pdf;
+using iText.Kernel.Pdf.Canvas.Parser;
+using iText.Kernel.Pdf.Canvas.Parser.Listener;
+using iText.Layout;
+using iText.IO.Font.Constants;
+using iText.Layout.Element;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5961168)